packages/Search/src/details/DTK_SearchMacros.hpp

```diff
@@ -0,0 +1,17 @@
+/****************************************************************************
+ * Copyright (c) 2012-2019 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef DTK_SEARCH_MACROS_HPP
+#define DTK_SEARCH_MACROS_HPP
+
+#define DTK_SEARCH_MARK_REGION( x ) std::string( "DTK_Search_" ) + x
```

Comment:
What is the upside of even having this macro now that we have reduced what it does to adding a simple prefix?

Reply:
So that you don't need to add this prefix manually each time and be consistent throughout.